pymysql/converters.py

```diff
@@ -89,9 +89,11 @@ def escape_time(obj):
     return escape_str(s)
 
 def escape_datetime(obj):
+    obj = obj.replace(tzinfo=None)
```

Comment:
I think engineer should know and care about it.
I don't want to do anything magical.